WickedEngine/wiScene_BindLua.cpp

```diff
@@ -701,6 +702,35 @@ int Scene_BindLua::Merge(lua_State* L)
 	return 0;
 }
 
+int Scene_BindLua::Entity_FindAllEntities(lua_State* L)
+{
+
+	int argc = wi::lua::SGetArgCount(L);
+	if (argc > 0)
+	{
+		wi::lua::SError(L, "Scene::Entity_FindAllEntities() doesnt take any argument");
+		return 0;
+	}
+	else {
+		wi::unordered_set<wi::ecs::Entity> listOfAllEntities;
+		scene->FindAllEntities(listOfAllEntities);
+		
+		int idx = 1; // lua indexes start at 1
+
+		lua_newtable(L); // build the return table
+		for (wi::ecs::Entity entity : listOfAllEntities)
+		{
+			wi::lua::SSetLongLong(L, entity);
+			lua_rawseti(L, -2, idx); // the table is now at -2 because we pushed the entity
```

Comment:
Why do you hardcode -2 here? Please see an example of populating a table in the function `Entity_GetNameArray()`, where idx of the table is queried by `lua_gettop`.